docs/competition/teams/role-descriptions/tinker.md

```diff
@@ -1,31 +0,0 @@
----
-original:
-  authors: Jeremy Morse, Rob Gilton and Sean Bedford
-  url: https://github.com/srobo/comp-team-coord-docs/blob/master/RoleDescriptions/tinker.md
-  extra: |
-    The original work is based on work done by Jeremy Morse, which was under the
-    Creative Commons Attribution 4.0 License.  To view a copy of this license,
-    visit <a href="https://creativecommons.org/licenses/by/4.0/">https://creativecommons.org/licenses/by/4.0/</a>.
----
-# Tinker Time Coordinator
```

Comment:
Why is this being dropped? Is there something to replace it? (Would have been great to have this described in the commit message)

Reply:
Yup, there's an existing page for this, which contains all the relevant information https://studentrobotics.org/runbook/competition/team-support/tinker-time/